LinqToRest/IResourceRetriever.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Messerli.LinqToRest
+{
+    public interface IResourceRetriever
+    {
+        T RetrieveResource<T>(Uri uri);
+
+        object RetrieveResource(Type type, Uri uri);
+    }
+}
```

Comment:
Add terminating newline

Reply:
Addressed in https://github.com/messerli-informatik-ag/linq-to-rest/pull/10/commits/f6bf1050eaeec4b5eadbc26c5304e13fa0470b0b

Reply:
> 
> 
> And extend changelog

Addressed in https://github.com/messerli-informatik-ag/linq-to-rest/pull/10/commits/69e58c8217304840c57d388036909a28c2f7031d and https://github.com/messerli-informatik-ag/linq-to-rest/pull/10/commits/4b0bc65a985451504e535fba0cfaa42e8bb518a8